### ISSUE TITLE
Julia, string in string interpolation

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments#877]
(Original issue 1170 created by innerlee on 2015-11-15T02:42:18.934217+00:00)

if a string interpolation contains string expressions, highlighting may leads confusion. e.g.,

```
#!julia
"n $(D > 100 ? " if D > 100 " : " else D <= 100 ")"
```

In the above string, color of string " if D > 100 " is misleading. It seems that "n $(D > 100 ? " is a string and " : " is a string and ")" is a string.

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2016-02-02T10:23:23.655892+00:00)

Fixed Julia string interpolation (closes #1170).

→ <<cset 4e7f70caea5b>>

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2016-02-02T10:23:34.381148+00:00)

Yes, this is a problem with interpolation syntax that can include arbitrary expressions, which we can't parse. I pushed a change to fix this in most cases.
